contracts/adapters/FeeSplitExtension.sol

```diff
@@ -19,30 +19,36 @@ pragma solidity 0.6.10;
 import { Address } from "@openzeppelin/contracts/utils/Address.sol";
 import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
 
-import { BaseAdapter } from "../lib/BaseAdapter.sol";
+import { BaseExtension } from "../lib/BaseExtension.sol";
 import { IIssuanceModule } from "../interfaces/IIssuanceModule.sol";
 import { IBaseManager } from "../interfaces/IBaseManager.sol";
 import { ISetToken } from "../interfaces/ISetToken.sol";
 import { IStreamingFeeModule } from "../interfaces/IStreamingFeeModule.sol";
 import { PreciseUnitMath } from "../lib/PreciseUnitMath.sol";
 import { TimeLockUpgrade } from "../lib/TimeLockUpgrade.sol";
+import { MutualUpgrade } from "../lib/MutualUpgrade.sol";
 
 
 /**
- * @title FeeSplitAdapter
+ * @title FeeSplitExtension
  * @author Set Protocol
  *
- * Smart contract adapter that allows for splitting and setting streaming and mint/redeem fees. 
+ * Smart contract extension that allows for splitting and setting streaming and mint/redeem fees.
  */
-contract FeeSplitAdapter is BaseAdapter, TimeLockUpgrade {
+contract FeeSplitExtension is BaseExtension, TimeLockUpgrade, MutualUpgrade {
     using Address for address;
     using PreciseUnitMath for uint256;
     using SafeMath for uint256;
 
     /* ============ Events ============ */
 
-    event FeesAccrued(address indexed _operator, address indexed _methodologist, uint256 _operatorTake, uint256 _methodologistTake);
-    
+    event FeesDistributed(
```

Comment:
Don't know if IC analytics is looking for the `FeesAccrued` event but should let them know this is changed if they are. Think it's definitely better naming